ONSdigital/babbage
# Feature/cv19 statements and letters

### What

- Added a new link to product pages - "All statements and letters" - which takes you to the statements and letters section of the site, with the first keyword included as a search query. This link is only rendered when the product in question has `keywords`.

- Extracted the "EnableCovid19Feature" string in the `PageRequestHandler` into its own const for consistency

### How to review

- Check code changes make sense
- Check that the copy aligns with requirements
- Load a product page with keywords, such as COVID-19. Check to see that the new link is present
- Check that clicking on the link takes you to the statements and letters search page with a single query string - the first keyword of the COVID-19 data
- Check that the link does not render on a page where no keywords are available in the data

### Who can review

Anyone but me
